client/gutenberg/extensions/markdown/jetpack-markdown-block-editor.jsx

```diff
@@ -82,37 +77,19 @@ class JetpackMarkdownBlockEditor extends Component {
 			}
 		};
 
-		// Manages css classes for each panel based on component's state
-		const classesForPanel = function( panelName ) {
-			return classNames( {
-				'components-tab-button': true,
-				'is-active': this.state.activePanel === panelName,
-				[ `${ className }__${ panelName }-button` ]: true,
-			} );
-		};
-
 		return (
 			<div className={ className }>
 				<BlockControls>
-					<ButtonGroup>
-						<button
-							className={ classesForPanel.call( this, 'editor' ) }
-							onClick={ this.showEditor }
-						>
-							<span>{ __( 'Markdown' ) }</span>
-						</button>
-						<button
-							className={ classesForPanel.call( this, 'preview' ) }
-							onClick={ this.showPreview }
-						>
-							<span>{ __( 'Preview' ) }</span>
-						</button>
-					</ButtonGroup>
+					<div className="components-toolbar">
```

Comment:
Why not use the `Toolbar` component? As usual, we should not be reusing classes, only components.

Reply:
Well, because the `<Toolbar />` will always want to render `<IconButton />`s - see https://github.com/WordPress/gutenberg/blob/20650111b6e307eef4dc2850d044f2be682c0871/packages/components/src/toolbar/index.js#L66

An option is to use `<ToolbarContainer />` (see https://github.com/WordPress/gutenberg/blob/55cac4e3168158e5a4e90eb0366844a7b7744fa1/packages/components/src/toolbar/toolbar-container.js#L6) - but it seemed that this is an internal API, so I wasn't sure it should be used.

What do you think?

Reply:
We should probably find a way to address this in Gutenberg components. Perhaps the right solution is to make the text button something supported in general?

_This is not a blocker for this, just something we should address_. 

cc @aduth @blowery as you were looking at buttons and component overlap.

Reply:
> Perhaps the right solution is to make the text button something supported in general?

That would be great! Honestly, I was surprised to see that we don't support that option out of the box.

Reply:
I'm suggesting we keep that as-is right now, and eventually if we get support for those text buttons, we can update it. It's good that we have a real life example usage in core blocks - the custom HTML block - we can strive to maintain consistency with what it does (which is what this PR suggests as well).

Reply:
Seems like it would be useful for core HTML block as well: https://github.com/WordPress/gutenberg/blob/f9e82d00668ec88a2cc355fb33e441f85393728e/block-library/html/index.js#L68-L83

Reply:
@simison see my comment above yours 😉

Reply:
:thumbsup: :thumbsup: yeah I was little late hitting "comment" :-D